src/aiida/plugins/utils.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
     def logger(self) -> Logger:
         return self._logger
 
-    def get_version_info(self, plugin: str | type) -> dict[t.Any, dict[t.Any, t.Any]]:
```

Comment:
@unkcpz FYI I think this has a similar issue that we discussed, I don't think adding Any here is correct. Let's discuss on the other PR, but just flagging it here as well so we don't forget.

Reply:
The original error was:

> src/aiida/plugins/utils.py:67: error: Subclass of "str" and "FunctionType" cannot exist: "FunctionType" is final  [unreachable]

Still struggling to understand, will need to experiment a bit.